action.yml

```diff
@@ -50,6 +50,9 @@ inputs:
   polaris_serverUrl:
     description: 'Polaris Server URL'
     required: false
+  bridge_polaris_triage:
+    description: 'Polaris Triage Option'
```

Comment:
Option is not needed

Reply:
Also can you take care of contract test and mock bridge changes ?

Reply:
@sig-tithi :  Can I add it as Polaris Triage Value or just Polaris Triage

Reply:
updated